reference/glossary.md

```diff
@@ -0,0 +1,17 @@
+(reference-glossary)=
+# Glossary
+
+```{glossary}
+
+Amazon Web Services (AWS)
+    A cloud platform provided by Amazon that can be used to host Charmed HPC. See the [AWS website](https://aws.amazon.com/) for more information.
+
+Graphics Processing Unit (GPU)
+    A specialized processor that is designed to accelerate image processing and graphics rendering for output to a display device.
+
+Juju
+    A charmed operator framework that helps you deploy, integrate and manage applications across multiple environments. 
```

Comment:
I wouldn't say `charmed operator` here as anyone new to Charmed HPC likely won't be familiar with Juju's nomenclature. I wordsmithed this suggested change below using content from `juju.is` that makes things a bit more high-level for those not immediately familiar with Juju:

```suggestion
    An open source orchestration engine that enables the deployment, integration and management of applications at scale on multiple cloud environments.
```

I know 'cloud' is an adverse term to use with HPC people, but Juju is limited to environments that behave like clouds. There needs to be an API that Juju can talk to and understand. It can't do things like PXE boot machines; it needs MAAS for that. Also, environments is a bit nebulous.